src/lib/options.ts

```diff
@@ -53,7 +52,7 @@ export type YahooFinanceOptions = Static<typeof YahooFinanceOptionsSchema> & {
 };
 
 const options: YahooFinanceOptions = {
-  YF_QUERY_HOST: process.env.YF_QUERY_HOST || "query2.finance.yahoo.com",
```

Comment:
This one was a bit painful as CloudFlare would require a `import * as process from "node:process"` here. But that won't work in a browser, seeing as this was one of very few instances of this I figured I could just delete it. Perhaps not, though?

Reply:
Yeah, I wouldn't just delete this :sweat_smile:  Also because that would be a breaking change.  I'll write some initial thoughts on `process.env` further below.